src/bundle/Resources/public/js/scripts/admin.location.view.js

```diff
@@ -8,18 +8,18 @@
     const queryString = window.location.search;
     const urlParams = new URLSearchParams(queryString);
     const publishedContentId = urlParams.get('publishedContentId');
-    const handleEditItem = (content, location) => {
+    const handleEditItem = (content, location, isLanguageSelectorOpened) => {
         const contentId = content._id;
         const locationId = location._id;
         const languageCode = content.mainLanguageCode;
         const checkVersionDraftLink = Routing.generate('ibexa.version_draft.has_no_conflict', { contentId, languageCode, locationId });
-        const submitVersionEditForm = () => {
+        const updateForm = () => {
             doc.querySelector('#form_subitems_content_edit_content_info').value = contentId;
             doc.querySelector(`#form_subitems_content_edit_language_${languageCode}`).checked = true;
-            doc.querySelector('#form_subitems_content_edit_create').click();
         };
         const addDraft = () => {
-            submitVersionEditForm();
+            updateForm();
+            doc.querySelector('#form_subitems_content_edit_create').click();
```

Comment:
maybe put this line into extra function? you use it three times, will be easier if for example field id would change :)